bot/controllers/verify.js

```diff
@@ -35,21 +40,27 @@ class VerifyController extends BaseController {
     const email = message.parsed[1].toLowerCase();
     const emailDomain = email.split('@').pop();
 
-    // We can set `codeLength` to whatever length we want the verif code to be.
+    // We can set `codeLength` to whatever length we want the verify code to be.
     // Recommend ngt 8 digits.
     if (validDomains.includes(emailDomain)) {
       const codeLength = 6;
       // code to equal value generated
       const code = generateCode(codeLength);
-
       util.log('code', code, 3);
       // TODO: Set `time` prop to 600000 (10min)
-      const collector = message.channel.createMessageCollector(
+      if (message.content === code) {
+        util.log('words');
+      }
+      const collector = client.author.message.createMessageCollector(
         m => m.content.includes(code),
         { time: timeoutInMiliseconds });
```

Comment:
This not your fault, but looks like Milliseconds is miss spelled here and in the variable name at the top of verifyAction.